### ISSUE TITLE
Intermediate checkpoints with ResNet backbone

### ISSUE THREAD

**Issue by ndrsn0208** [mlfoundations/open_clip#1009]
I found the intermediate checkpoints for the ViT based models. Are there any ResNet based checkpoints available? Thank you!

**Comment by rwightman** [comment]
@ndrsn0208 I don't think we have those.  There are some laion2b convnext ones kicking around I could probably dig up.
